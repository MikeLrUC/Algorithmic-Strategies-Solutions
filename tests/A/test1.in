5
3 5
2 2 4
0 0 4
4 0 0
3 6
4 2 2
4 0 4
2 2 2
3 3
2 0 0
0 2 0
4 4 0
4 6
0 0 0 2
4 0 0 2
2 0 2 0
4 0 8 8
12 30
0 0 0 8 4 0 2 0 2 0 0 2
2 4 0 8 16 2 16 8 8 2 0 2
2 2 4 0 0 0 2 0 0 2 4 4
4 4 2 2 8 8 2 2 0 0 0 0
0 0 0 0 0 0 0 0 0 0 0 0
2 0 0 2 0 0 2 0 0 2 0 2
4 2 0 0 4 2 2 0 0 8 16 2
0 0 2 0 0 0 2 0 0 0 0 0
0 0 0 0 0 2 0 0 0 0 0 0
0 0 2 2 0 2 4 8 16 2 0 0
4 2 4 2 4 2 4 2 4 2 4 2
0 2 2 8 8 4 2 4 2 0 0 2
