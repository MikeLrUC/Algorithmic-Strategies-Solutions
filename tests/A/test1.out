4
no solution
no solution
5
3
